docs/UserGuide.md

```diff
@@ -94,6 +94,9 @@ that you key in. Any success, error or warning messages will be displayed in thi
 
 Every item is made up of an item name, location and quantity. Expiry date and tags are optional. 
 
+Expiry date of an item will be shown in orange if the item is expiring in 3 days. Expiry date of an item will be shown
```

Comment:
I think this should be "expiring within the next 3 days".

Reply:
will make the change :)